ci-operator/step-registry/osd-ccs/fleet-manager/test/osd-ccs-fleet-manager-test-commands.sh

```diff
@@ -0,0 +1,357 @@
+  exit 1
+fi
+
+echo "Confirming that ServiceAccount for cluster-proportional-autoscaler was created"
+# check that cluster-proportional-autoscaler ServiceAccount was created
+CL_PROP_AUTOSCALER_SA=$(oc get ServiceAccount -A | grep -c cluster-proportional-autoscaler)
+if [ "$CL_PROP_AUTOSCALER_SA" -lt 1 ]; then
+  echo "cluster-proportional-autoscaler Service Accounts not found. Exiting..."
+  exit 1
+fi
+
+# confirm that the default PriorityClass has GLOBAL-DEFAULT flag set to true and VALUE = 0
+DEFAULT_PRIORITY_CLASS=$(oc get PriorityClass -A | grep default | awk '{print $2,$3}')
+
+echo "Confirming that 'default' PriorityClass has GLOBAL-DEFAULT set to true and value = 0"
+if [[ "$DEFAULT_PRIORITY_CLASS" != *"true"* ]] || [[ "$DEFAULT_PRIORITY_CLASS" != *"0"* ]];then
+  echo "'default' PriorityClass should have value 0 and GLOBAL-DEFAULT set to true. Got the value and GLOBAL_DEFAULT: $DEFAULT_PRIORITY_CLASS. Exiting..."
+  exit 1
+fi
+
+# confirm that the default PriorityClass has GLOBAL-DEFAULT flag set to true and VALUE = 0
+OVERPROVISIONING_PRIORITY_CLASS=$(oc get PriorityClass -A | grep overprovisioning | awk '{print $2,$3}')
+
+echo "Confirming that 'overprovisioning' PriorityClass has GLOBAL-DEFAULT set to false and value = -1"
+if [[ "$OVERPROVISIONING_PRIORITY_CLASS" != *"false"* ]] || [[ "$OVERPROVISIONING_PRIORITY_CLASS" != *"-1"* ]];then
+  echo "'overprovisioning' PriorityClass should have value -1 and GLOBAL-DEFAULT set to false. Got the value and GLOBAL_DEFAULT: $OVERPROVISIONING_PRIORITY_CLASS. Exiting..."
+  exit 1
+fi
+
+###### end of proportional autoscaler tests (OCP-63511) ######
+
+##############################################################
+
+###### disable workload monitoring tests (OCP-60338) ######
+
+function check_monitoring_disabled () 
+{
+  echo "Checking workload monitoring disabled for $1"
+  # should be more than 0
+  DISABLED_MONITORING_CONFIG_COUNT=$(oc get configmap cluster-monitoring-config -n openshift-monitoring -o yaml | grep -c "enableUserWorkload: false")
+  if [ "$DISABLED_MONITORING_CONFIG_COUNT" -lt 1 ]; then
+    echo "Workload monitoring should be disabled by default. Exiting..."
+    exit 1
+  fi
+}
+
+## check workload monitoring disabled on a service cluster
+
+export KUBECONFIG="${SHARED_DIR}/hs-sc.kubeconfig"
+check_monitoring_disabled "service cluster"
+
+## check workload monitoring disabled on a management cluster
+
+export KUBECONFIG="${SHARED_DIR}/hs-mc.kubeconfig"
+check_monitoring_disabled "management cluster"
+
+###### end of disable workload monitoring tests (OCP-60338) ######
+
+##################################################################
+
+###### Sector predicates to support multiple sectors by labels tests (OCP-63998) ######
+
+sc_cluster_id=$(cat "${SHARED_DIR}"/osd-fm-sc-id)
+mc_cluster_id=$(cat "${SHARED_DIR}"/osd-fm-mc-id)
+
+INITIAL_MC_COUNT=$(ocm get /api/osd_fleet_mgmt/v1/management_clusters  --parameter search="region is 'ap-northeast-1'" | jq -r .total)
```

Comment:
Maybe we can define OSDFM_REGION=${LEASED_RESOURCE} like others, in theory the region can be customized in the future.

Reply:
done